### ISSUE TITLE
rtransfer_link-0.1.0 continously hangs with unsupported arguments

### ISSUE THREAD

**Issue by AJRubio-Montero** [onedata/onedata#41]
Good evening.

I'm testing clean deployments of OneProvider based on 20.02.4 as well as 19.02.5 releases, and the *rtransfer_link-0.1.0* service is continously hanging due to be launched with unsupported parameters.

This is the continous entry in  *root/var/log/op_worker* . Although this is obtained from the 19.02 release, the ones belonging to 20.02.04 are similar:

```log
[I 13:17:02.554 <0.2547.11>] Starting port  /opt/onedata/onedata1902/root/usr/lib64/op_worker/lib/rtransfer_link-0.1.0/priv/link --v=0 --log_dir=/opt/onedata/onedata1902/root/var/log/op_worker/link --log_link=/opt/onedata/onedata1902/root/var/log/op_worker --server_port=6665 --single_fetch_max_size=12582912 --graphite_namespace_prefix=rtransfer_link --graphite_reporting_period=60 --max_open_descriptors=1000 --descriptor_cache_duration=60000 --descriptor_cache_tick=1000 --number_of_data_conns=16 --recv_buf_size=8388608 --send_buf_size=8388608 --use_ssl --ssl_cert_path=/tmp/tmp.CM4K6kEP3m --ssl_key_path=/opt/onedata/onedata1902/root/etc/op_panel/certs/web_key.pem --max_incoming_buffered_size=20971520 --storage_buckets=100 --send_congestion_flavor=bbr --shaper_quantum_ms_size=50 --helper_workers=100 --webdav_helper_workers=25 --throughput_probe_interval=25
ERROR: unknown command line flag 'log_dir'
ERROR: unknown command line flag 'log_link'
ERROR: unknown command line flag 'v'
[E 13:17:02.671 <0.2547.11>] link port exited with 1
[E 13:17:02.671 <0.626.0>] Supervisor rtransfer_link_sup had child rtransfer_link_port started with rtransfer_link_port:start_link() at <0.2547.11> exit with reason {shutdown,link_port_down} in context child_terminated
```
 
If you manually debug it, logging into the *op_worker* account, sourcing the */opt/onedata/onedataxxx/enable* profile and then executing above command, it reports as unsupported these parameters: *log_dir*, *log_link* and *v*.  When you remove them, the command seems to work.

The only way that I provisionally solved the issue is using a Bash wrapper, which removes these arguments. For example for the 20.02 release:

```bash
$
$ cd /opt/onedata/onedata2002/root/usr/lib64/op_worker/lib/rtransfer_link-0.1.0/priv
$ cat link.wrapper

#!/bin/bash

ARGS=()
for var in "$@"; do
   # Ignore bad arguments
   [[ "$var" == --v=* ]] && continue
   [[ "$var" == --log\_dir=* ]] && continue
   [[ "$var" == --log\_link=* ]] && continue

   ARGS+=("$var")
done

/opt/onedata/onedata2002/root/usr/lib64/op_worker/lib/rtransfer_link-0.1.0/priv/link.bin  "${ARGS[@]}"

$
$ cp link link.bin
$ cp link.wrapper link
$ chmod +x link
$ # reboot or systemctl restart op_worker
```

I remember this behaviour ocasionally appeared in previous releases and it could mask other kind of errors/bugs/missconfigurations.  

Can you please give us some support?
(The behaviour is easily reproducible if you install an OneProvider from the CentOs 7 repos).

Thank you in advance.
Happy new year.

**Comment by bkryza** [comment]
@AJRubio-Montero 
Did you install the Oneprovider using packages or using onedatify and Docker containers?

**Comment by AJRubio-Montero** [comment]
Hello @bkryza 

I installed using packages.

Cheers.

**Comment by bkryza** [comment]
@AJRubio-Montero This was an issue related to bad linking with gflags library on CentOS - this is now fixed since version 20.02.6...
